pages/bitmark-references/bitmark-sdk/README.md

```diff
@@ -4,74 +4,75 @@
+    * [Proposing a transfer offer](action.md#proposing-a-transfer-offer)
+    * [Querying for offers](action.md#querying-for-offers)
+    * [Accepting a transfer offer](action.md#accepting-a-transfer-offer)
+    * [Rejecting a transfer offer](action.md#rejecting-a-transfer-offer)
+    * [Canceling a transfer offer](action.md#canceling-a-transfer-offer)
 
 
-## [Query](query.md#query)
+## [Queries](query.md)
 
 * [Asset](query.md#asset)
   * [Record data structure](query.md#record-data-structure)
-  * [Query for a specific asset](query.md#query-for-a-specific-asset)
-  * [Query for a set of assets](query.md#query-for-a-set-of-assets)
-    * [assets registered by the specific registrant](query.md#assets-registered-by-the-specific-registrant)
+  * [Querying for a specific asset](query.md#querying-for-a-specific-asset)
+  * [Querying for a set of assets](query.md#querying-for-a-set-of-assets)
+    * [Assets registered by the specific registrant](query.md#assets-registered-by-the-specific-registrant)
 * [Transaction (tx)](query.md#transaction-tx)
   * [Record data structure](query.md#record-data-structure-1)
-  * [Query for a specific transaction](query.md#query-for-a-specific-transaction)
-  * [Query for a set of transactions](query.md#query-for-a-set-of-transactions)
-    * [The provenance of a bitmark](query.md#the-provenance-of-a-bitmark)
-    * [The transaction history of an account](query.md#the-transaction-history-of-an-account)
+  * [Querying for a specific transaction](query.md#querying-for-a-specific-transaction)
+  * [Querying for a set of transactions](query.md#querying-for-a-set-of-transactions)
+    * [Transactions depicting provenance of a bitmark](query.md#transactions-depicting-the-provenance-of-a-bitmark)
+    * [Transactions created by an account](query.md#transactions-created-by-an-account)
 * [Bitmark](query.md#bitmark)
   * [Record data structure](query.md#record-data-structure-2)
-    * [offer](query.md#offer)
-  * [Query for a specific bitmark](query.md#query-for-a-specific-bitmark)
-  * [Query for a set of bitmarks](query.md#query-for-a-set-of-bitmarks)
+    * [Offer data structure](query.md#offer-data-structure)
+  * [Querying for a specific bitmark](query.md#querying-for-a-specific-bitmark)
+  * [Querying for a set of bitmarks](query.md#querying-for-a-set-of-bitmarks)
     * [bitmarks issued by the specific issuer](query.md#bitmarks-issued-by-the-specific-issuer)
     * [bitmarks owned by the specific owner](query.md#bitmarks-owned-by-the-specific-owner)
     * [bitmarks offered from the specific sender](query.md#bitmarks-offered-from-the-specific-sender)
     * [bitmarks offered to the specific receiver](query.md#bitmarks-offered-to-the-specific-receiver)
 
-
-## [Store Seed](store-seed.md#store-seed)
-  
-* [Android](store-seed.md#android)
-* [iOS](store-seed.md#ios)
-
-
-## [Web Socket](websocket.md#web-socket)
+## [Triggering from Web Socket Events](websocket.md#web-socket)
```

Comment:
```suggestion
## [Subscribing to events via WebSocket](websocket.md#web-socket)
```

The server triggers events to notify clients about the updates on the blockchain. For SDK users, the action should be about how to get the events sent from the server.